docs/deploy-full-rp-service-in-dev.md

```diff
@@ -247,9 +263,24 @@
         --service-endpoints Microsoft.ContainerRegistry
     ```
 
+1. Register your subscription with the resource provider (post directly to subscription cosmosdb container)
+    ```bash
+    curl -k -X PUT   -H 'Content-Type: application/json'   -d '{
+        "state": "Registered",
+        "properties": {
+            "tenantId": "'"$AZURE_TENANT_ID"'",
+            "registeredFeatures": [
+                {
+                    "name": "Microsoft.RedHatOpenShift/RedHatEngineering",
+                    "state": "Registered"
+                }
+            ]
+        }
+    }' "https://localhost:8443/subscriptions/$AZURE_SUBSCRIPTION_ID?api-version=2.0"
+    ```
+
 1. Create the cluster
     ```bash
-    export NO_INTERNET=true
```

Comment:
if you're using the az-cli to create the cluster (which the docs are), this flag doesn't take affect, hence the removal